### DISCUSSION TITLE
tuya TS0202 PIR remains occupied

### DISCUSSION THREAD

**Discussion by wjansenw** [Koenkk/zigbee2mqtt#16577] [Devices]
Hi,
I have a TuYa PIR sensor, but it keeps being flagged as occupied.

https://www.zigbee2mqtt.io/devices/ZM-35H-Q.html#tuya-zm-35h-q (this is the model according to the package, but it looks a bit different)
this is what I ordered: https://nl.aliexpress.com/item/1005004776127380.html?spm=a2g0o.order_list.order_list_main.18.21ef79d2OFLyEL&gatewayAdapt=glo2nld

What I see in zigbee2mqtt:
![afbeelding](https://user-images.githubusercontent.com/25421739/217353611-2402bb09-5ccb-434b-9e6f-19c6d4a96825.png)

I run zigbee2mqtt as HA add-on; using a conbee 2.

Willing to help in testing, providing log files etc, but still finding out the basics of HA and zigbee.

Tnx,
Wim

**Comment by Koenkk** [comment]
- Could you provide the data/database.db entry of this device?
- Can you provide the debug log when triggering occupancy + 5 minutes after this?

See https://www.zigbee2mqtt.io/guide/usage/debug.html on how to enable debug logging.

**Reply by leonidostrovski** [reply]
I have exactly the same issue with different types of sensors, and I think many people on the internet search for solutions to the same issue.
If I understand correctly, the issue is very simple. The zigbe2mqtt displays correctly the last status received from the sensor, regardless of the time since the last status was received. 
If, due to any reason, the update "clear" from the sensor to zigbe2mqtt has not arrived, the zigbe2mqtt will continue to see the sensor  in the occupied state.

I am looking for any way to set the zigbee2mqtt to reset status to sensor if no data arrives from it after x time.

It will not resolve the root cause of the problem, but I think it will solve it.

**Comment by wjansenw** [comment]
Actually, I think the device is broken. The 2nd one seems to report states
correctly.

I'll continue to test.

Tnx,
Wim

Op wo 8 feb. 2023 16:52 schreef Koen Kanters ***@***.***>:
